pkgs/development/python-modules/pyunormalize/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pyunormalize";
+  version = "15.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-z0qHRRoPHLdpEaqX9DL0V54fVkovDITOSIxzpzkBtsE=";
+  };
+
+  doCheck = true;
```

Comment:
Isn't this the default?

Reply:
Yes, it seems so. I wasn't sure about it myself, and I thought it would be better for the maintainers to not rely on implicit knowledge. I could also remove it, it doesn't seem to be a very unified thing in the code base:

```
$ git grep 'doCheck = true' pkgs/development/python-modules| wc -l
94
```

Reply:
Please remove it.

Reply:
okay, removed.